### PR TITLE
ENCD-4697 add biosample summary to experiment series

### PR DESCRIPTION
**Pull request by yunhailuo**

### CODE REVIEW

File: src/encoded/static/components/experiment_series.js [hunk 1]
```diff
@@ -171,12 +169,13 @@ const ExperimentSeriesComponent = (props, reactContext) => {
...
 
-                                {terms.length > 0 || speciesRender ?
+                                {(context.biosample_summary && context.biosample_summary.length > 0) || speciesRender ?
                                     <div data-test="biosamplesummary">
                                         <dt>Biosample summary</dt>
                                         <dd>
-                                            {terms.length > 0 ? <span>{terms.join(' and ')} </span> : null}
-                                            {speciesRender ? <span>({speciesRender})</span> : null}
+                                            {speciesRender ? <span>{speciesRender}</span> : null}
+                                            &nbsp;
```
Comment (forresttanaka) [inline src/encoded/static/components/experiment_series.js]: This inserts a non-breaking space even if `speciesRender` is null/undefined. That would be visible as an odd space before the `biosample_summary` in that case. Like before, you can just put a space before the previous `</span>` and it should work in all cases.

biosample summary

Comment (yunhailuo) [reply src/encoded/static/components/experiment_series.js]: Thanks

**Comment by yunhailuo** [comment]
One test added.

File: src/encoded/tests/test_types_experiment_series.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    res = testapp.get(base_experiment_series['@id']+'@@index-data')
+    assert res.json['object']['biosample_summary'] == [
+        'S2R+ nuclear fraction treated with ethanol'
+    ]
+    testapp.patch_json(
+        base_experiment_series['@id'],
+        {'related_datasets': [experiment_1['@id'], experiment_2['@id']]}
+    )
+    res = testapp.get(base_experiment_series['@id']+'@@index-data')
+    print(res.json['object']['biosample_summary'])
```
Comment (gabdank) [inline src/encoded/tests/test_types_experiment_series.py]: I think @yunhailuo you need to remove the print statement

Comment (yunhailuo) [reply src/encoded/tests/test_types_experiment_series.py]: Sorry, that's really bad. Fixed.

**Pull request merged**